qt/python/mantidqtinterfaces/mantidqtinterfaces/TexturePlanner/view.py

```
@@ -0,0 +1,779 @@
+            angs.setMinimum(-360)
+            angs.setMaximum(360)
+        for angs in self.init_angles:
+            angs.setMinimum(-360)
+            angs.setMaximum(360)
+
+    def set_translation_step(self, step_size: float) -> None:
+        self.spnInitPX.setSingleStep(step_size)
+        self.spnInitPY.setSingleStep(step_size)
+        self.spnInitPZ.setSingleStep(step_size)
+
+    def set_translation_limits(self, min: float, max: float) -> None:
```

Ruff (0.15.21)

[error] Function argument `min` is shadowing a Python builtin

(A002)